Assets/Scenes/editor.unity

```diff
@@ -2369,6 +2370,10 @@ MonoBehaviour:
   NumberInputAtom: {fileID: 5812454901384158741, guid: 643cb13cbab094b42887dae813c93af2, type: 3}
   BooleanInputAtom: {fileID: 5233136792266491960, guid: 42f6030112784864f9dc60d33b26df8a, type: 3}
   Vector3InputAtom: {fileID: 5341960638517146307, guid: 45f07c32725cf3c419e1758b1f6b2ab0, type: 3}
+  ContextMenuAtom: {fileID: 9033582606962256507, guid: 8db363346bbc09646b96fffac6224be5, type: 3}
```

Comment:
Context menu atoms are not assigned in the Unity State

![grafik](https://user-images.githubusercontent.com/11379989/203119366-ac8244d5-a699-4f4a-9e24-2cae1ef3175d.png)

Reply:
Never mind. This was caused by some Checkout problems on my side